### PR TITLE
[PS] Fixes to medical release v2

### PR DESCRIPTION
**Pull request by MocanaAtBroad**

### CODE REVIEW

File: pepper-apis/studybuilder-cli/studies/singular/patch-log.conf [hunk 1]
```diff
@@ -3,7 +3,6 @@
     "SingularPatientSurveyUpdate",
     "SingularUpdates",
     "SingularAboutHealthyPexTask",
-    "SingularUpdateConsentCopyEvents",
     "SingularMakeNoMedicalTreatmentOptionExclusive",
     "SingularUpdateConsentCopyEvents"
     "SingularFixRaceQuestionTypo"
```
Comment (MocanaAtBroad) [inline pepper-apis/studybuilder-cli/studies/singular/patch-log.conf:8]: Not directly part of PR objective, but there was an entry in patch-log.conf that was duplicated and messing up applying the patches.

File: pepper-apis/studybuilder-cli/studies/singular/i18n/en.conf [hunk 2]
```diff
@@ -1660,15 +1660,15 @@
...
         "help_me_get_via_portal": "I would like the Project Singular team to assist me by phone or Zoom call, walking me through obtaining the necessary records via my online patient portal. I understand that someone from Boston Children’s Hospital will reach out to me to schedule a time that is convenient for me to meet with them by phone or Zoom.",
         "upload": "I will upload the records myself via the Medical Record Upload button on my Project Singular Dashboard. This will include my most recent cardiology clinic note, echo report, catheterization report, and operation notes."
-        "fax": "I will fax or email them to the team at Boston Children’s Hospital, fax to (617) 730-6201 or email to [BCH Project Singular Email]. This will include my most recent cardiology clinic note, echo report, catheterization report, and operation notes."
+        "fax": "I will fax or email them to the team at Boston Children’s Hospital, fax to (617) 730-6201 or email to <a href=\"mailto:contact@projectsingular.org\" class=\"Link\">contact@projectsingular.org</a>. This will include my most recent cardiology clinic note, echo report, catheterization report, and operation notes."
```
Comment (MocanaAtBroad) [inline pepper-apis/studybuilder-cli/studies/singular/i18n/en.conf:1663]: Missed including an actual email link

File: pepper-apis/studybuilder-cli/studies/singular/i18n/en.conf [hunk 3]
```diff
@@ -1607,10 +1607,10 @@
...
       "p2": "To proceed with our study, we are asking you to release $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child's\", \"your\") relevant medical records to our staff. First, you must sign this medical release form so that our study staff may obtain copies of $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child's\", \"your\") medical records, securely store them, and use them for this study. ",
       "p3": "Relevant medical records include operation notes, last cardiology clinic note, echo reports, and catheterization reports.",
-      "p4": "For our staff to receive your records, you have several options. Please select the option that you would like to use:",
+      "p4": "For our staff to receive $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child's\", \"your\") records, you have several options. Please select the option that you would like to use:",
```
Comment (MocanaAtBroad) [inline pepper-apis/studybuilder-cli/studies/singular/i18n/en.conf:1614]: Missed a few of these "your child" vs "your" in the text. I hope I got them all now.

File: pepper-apis/studybuilder-cli/studies/singular/i18n/en.conf [hunk 4]
```diff
@@ -1619,16 +1619,16 @@
...
+      "titleText": "By completing this information, you are agreeing to allow us to receive $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child’s\", \"your\") medical records through the option you selected above (sending in $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child’s\", \"your\") records via email, fax, or upload; working with Project Singular study staff to help you access $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child’s\", \"your\") records); or having our study staff contact $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child's\", \"your\") designated physician(s) and hospital(s)/institutions(s) to obtain your $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child’s\", \"your\")).",
       "p2": "By signing this form, I am acknowledging that:",
```
Comment (MocanaAtBroad) [inline pepper-apis/studybuilder-cli/studies/singular/i18n/en.conf:1632]: I had put the p1 text as, well, a separate paragraph. Request was to have with same font as title. It is done.

File: pepper-apis/studybuilder-cli/studies/singular/patches/singular-new-events.conf [hunk 5]
```diff
@@ -1,5 +1,34 @@
...
+        ]
+      },
+      "preconditionExpr": """
+        !user.studies["singular"].forms["MEDICAL_RECORD_RELEASE"].instances[latest].hasPreviousInstance() &&
+        user.studies["singular"].forms["CONSENT_DEPENDENT"].hasInstance()
+      """,
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 3
+    },
```
Comment (MocanaAtBroad) [inline pepper-apis/studybuilder-cli/studies/singular/patches/singular-new-events.conf:31]: For adult dependent medical release there was one dob field in last page that we had not copied the dob.
Here it is now.
I considered doing this in separate patch, but this one patch is already trying to add a bunch of events so seemed easier to do it this way.

**Review by ssettipalli — APPROVED** [review]

**Pull request merged**